tests/tools/src/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#Detray library, part of the ACTS project(R& D line)
+#
+#(c) 2024 CERN for the benefit of the ACTS project
+#
+#Mozilla Public License Version 2.0
+
+# C++17 support for CUDA requires CMake 3.18.
+cmake_minimum_required(VERSION 3.18)
+
+# Enable CUDA as a language.
+enable_language(CUDA)
+
+# Set the CUDA build flags.
+include(detray-compiler-options-cuda)
```

Comment:
I don't see any CUDA source files in this directory... :thinking:

Reply:
Without ```enable_language(CUDA)``` the linking seems to fail. Does that make sense?

Reply:
Ahh, interesting... :thinking:

This is indeed an interesting one. Since you currently compile the CUDA code of this directory's executable into a `STATIC` library, the CUDA runtime library is not getting linked into that one.

But then, when you use the `STATIC` library, you indeed need to activate CUDA as a language, otherwise CMake doesn't link the library correctly.

Okay, then let's keep CUDA enabled in this directory. That indeed seems like the easiest way to make this work. :thinking: